packages/arb-validator-core/ethbridge/arbRollupWatcher.go

```diff
@@ -227,7 +311,7 @@ func (vm *ethRollupWatcher) processMessageDeliveredEvents(chainInfo arbbridge.Ch
 			Message:   msg,
 		}, nil
 
-	} else if ethLog.Topics[0] == depositERC20MessageDeliveredID {
+	} else if ethLog.Topics[0] == depositERC20ID {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)